controllers/topics.controllers.js

```diff
@@ -0,0 +1,7 @@
+const { requireTopics } = require("../models /topics.model");
+
+exports.getTopics = (req, res) => {
+  requireTopics().then((data) => {
+    res.status(200).send(data.rows);
```

Comment:
This should be an object with a key of topics, not just the array. Trust me, you'll thank us in front end.